src/pages/admin/pages/faq/FaqPage.vue

```diff
@@ -3,16 +3,18 @@
     <va-card class="mb-8">
       <va-card-title> Do you have any questions? </va-card-title>
       <va-card-content>
-        <va-input class="m-3">
+        <va-input v-model="searchValue" class="m-3">
           <template #prepend>
             <va-icon name="search" />
           </template>
         </va-input>
       </va-card-content>
     </va-card>
+    <PopularCategories :categories="filteredCategories" />
+    <CTA />
 
     <va-card>
-      <va-card-title> Frequently Asked Questions </va-card-title>
+      <va-card-title> <h2>Popular questions</h2> </va-card-title>
```

Comment:
<img width="409" alt="image" src="https://github.com/epicmaxco/vuestic-admin/assets/5394573/bfa31d7d-6eaa-4fd6-bb3b-d3dcf03547e8">

Please stick to design (communicate you have to wait for collapses etc - you can do so in discord).

Reply:
1.8.0 is up by the way, so you can follow up with recent design.